range.c

```diff
@@ -1880,13 +1983,40 @@ range_alloc(VALUE klass)
 
 /*
  *  call-seq:
- *     range.count                 -> int
- *     range.count(item)           -> int
- *     range.count { |obj| block } -> int
+ *    count -> integer
+ *    count(&method) -> integer
```

Comment:
This line should be removed.  This passes a block created from `method` instead of a literal block, but behavior will be exactly the same.

Reply:
Done.